Makefile.gen.cluster_app.mk

```diff
@@ -18,6 +19,11 @@ validate-schema: ## Validate the JSON schema
 	go install github.com/giantswarm/schemalint/v2@v2
 	schemalint verify $(VALUES_SCHEMA) --rule-set=cluster-app
 
+.PHONY: generate-docs
+generate-docs: ## Generate values documentation from schema
+	go install github.com/giantswarm/schemadocs@latest
+	schemadocs generate $(VALUES_SCHEMA) -o $(CHART_README)
```

Comment:
I am doing something similar for my toy project using a gh action https://github.com/jkremser/log2rbac-operator/blob/master/.github/workflows/helm_docs.yaml it uses the values.yaml itself as a source of truth though.

Reply:
For cluster apps, we decided that it's much simpler to generate the docs from the schema, as this is more structured than the values.yaml, and contains everything. In cluster.apps, values.yaml only contains propertys that have a default value.